open-api/lang-parser/src/main/java/io/deephaven/lang/parse/CompletionParser.java

```diff
@@ -18,6 +19,7 @@
  */
 public class CompletionParser implements CompletionParseService<ParsedDocument, ChangeDocumentRequest.TextDocumentContentChangeEvent, ParseException> {
 
+    Logger LOGGER = LoggerFactory.getLogger(CompletionParser.class);
```

Comment:
`private static final`?

Reply:
ah, yes.  I thought I _did_ do that.  either way, it's otw.